src/main.cpp

```diff
@@ -338,6 +339,63 @@ install_specs(const std::vector<std::string>& specs, bool create_env = false)
     trans.execute(prefix_data, ctx.root_prefix / "pkgs");
 }
 
+void
+list_packages()
+{
+    auto& ctx = Context::instance();
+    PrefixData prefix_data(ctx.target_prefix);
+    prefix_data.load();
+
+    std::multimap<std::string, std::string, std::less<std::string>> ordered_packages;
+
+    std::cout << "List of packages in environment: " << ctx.target_prefix << std::endl;
+
+    //order list of packages from prefix_data by alphabetical order
+    for (auto package : prefix_data.m_package_records)
```

Comment:
Nice work! 

I wonder if it wouldn't be better to either use a std::tuple or std::array as a container for the 4 elements, or as another very nice option to use a struct.

you can make a new struct definition and I think it can be local to the function as well, e.g.


struct formatted_pkg
{
	std::string name, version, channel;
}

and then have the map be std::map<std::string, formatted_pkg> etc.